### ISSUE TITLE
Game log "immediatly-killed-after-combat" stores the unit names instead of the unit ids

### ISSUE THREAD

**Issue by Longwelwind** [Longwelwind/swords-and-ravens#340]
It's not top priority, but this is too make it more consistent with the other game logs. It will also make it future proof in case we change the name of a unit type in the future.

**Comment by gereon77** [comment]
Sorry -_- did it cause a crash?

**Comment by Longwelwind** [comment]
Don't worry; it didn't cause any crash at all.

It's not super important (and that's why it's low priority). Once the migration system is in place, it won't be hard to convert those logs, anyway.

**Comment by gereon77** [comment]
Ok. I will review all game logs in time because I guess there are other places where I have done that wrong also, like for houses, etc.

**Comment by gereon77** [comment]
Additional logs to fix:

- `killed-after-combat`

**Comment by gereon77** [comment]
Grabbed

**Comment by gereon77** [comment]
@Longwelwind Can be closed
